apps/vaporgui/AnnotationEventRouter.h

```diff
@@ -151,6 +149,8 @@ protected slots:
 
     AnimationParams *_ap;
     bool             _animConnected;
+
+    PGroup *_aa;
```

Comment:
Maybe a more descriptive name than `_aa`?

Reply:
Sounds good. Will do and note when it's done.